forte/data/data_store.py

```diff
@@ -187,6 +197,76 @@ def _new_tid(self) -> int:
         r"""This function generates a new ``tid`` for an entry."""
         return uuid.uuid4().int
 
+    def _get_type_info(self, type_name: str) -> Dict[str, int]:
+        """
+        Add a new type input_entry_class into ``_type_attributes``.
```

Comment:
docstring is off (not really `add a new type`). The main purpose is to get the type, `add a type` is kinda a side-effect (and the user don't need to know that something is added).